qemu/tests/cfg/live_snapshot_chain.cfg

```diff
@@ -59,3 +35,10 @@
             check_base_image = yes
             operate_timeout = 10
             check_alive_cmd = ""
+        - long_chain:
+            need_reboot = yes
+            file_create = yes
+            check_base_image_image1 = yes
+            check_base_image_sn64 = yes
```

Comment:
I think better set this param in the code instead of cfg file, since 64 is a configurable number. It should like /"check_base_image_sn%s", snapshot_num/